templates/zerver/help/unsubscribe-from-a-stream.md

```diff
@@ -19,3 +19,7 @@ You can always unsubscribe from any stream in Zulip.
 1. Click on the green checkmark the left of the stream name.
 
 {end_tabs}
+
+## Related topics
```

Comment:
This should be "Related articles" - we standardized on that label for these sections (previously, 4 were "Related topics").

Reply:
Okay I've updated the PR and changed Related topics to Related articles.